fetch/nosniff/resources/js.py

```diff
@@ -1,17 +1,19 @@
+from wptserve.utils import isomorphic_decode
```

Comment:
Looks like `isomorphic_decode` is imported but never used here.

Reply:
Updated. I should have run pylint before submit the code...